src/lib/converter.ts

```diff
@@ -0,0 +1,434 @@
+          .map(([key, value]) => {
+            const ctx = value as OpenAPIV3.ResponseObject
+            return ctx.content ? Object.keys(ctx.content) : []
+          })
+          .flat()
+        responses = accepts.length > 0 ? accepts : '*/*'
+      }
+
+      if (requestContext.servers) {
+        servers = requestContext.servers
+      }
+
+      if (requestContext.security) {
+        security = requestContext.security
+      }
+
+      return {
+        path,
+        group,
+        method,
+        summary,
+        description,
+        requestBody,
+        parameters,
+        responses,
+        servers,
+        security,
+      } as any
+    })
+
+    return ctx
+  }
+
+  /**
+   * @method createRequest
+   * @summary
+   *
+   * @param {Object} item
+   * @param {Number} index
+   * @param {Array<any>} array
+   */
+  private createRequest(item: any, index?: number, array?: any[]): void {
+    const document = this.apiParser.api as OpenAPIV3.Document
+    const { title } = document.info
+
+    const request = this.context.createRequest(
+      item.summary || item.path,
+      item.method,
+      new DynamicString(),
+      item.description,
+    )
+
+    if (item.requestBody) {
+      this.setRequesBody(
+        request,
+        item.requestBody as OpenAPIV3.RequestBodyObject,
+      )
+    }
+
+    if (item.parameters) {
+      this.setRequestParameters(item.parameters, request)
+    }
+
+    if (item.security) {
+      this.setRequestAuth(item.security, request)
+    }
+
+    if (item.responses) {
+      request.setHeader('Accepts', item.responses[0])
+    }
+
+    const requestURL = new PawURL(
+      document.paths[item.path] as OpenAPIV3.PathItemObject,
+      document,
+      item.path,
+      this.getEnviroment(),
+      request,
+    )
+
+    request.url = requestURL.fullUrl
+    this.requestGroups[item.group].appendChild(request)
+
+    if (requestURL.serverVariables) {
+      const envManager = this.envManagers[title]
+      const serverVars = requestURL.serverVariables
+
+      // Object.entries(serverVars).forEach(([varName, variable]) => {
+      //   const value = variable.default
+      //   envManager.setEnvironmentVariableValue(varName, value)
+      //   request.addVariable(
+      //     varName,
+      //     envManager.getDynamicString(varName),
+      //     variable.description || '',
+      //   )
+      // })
+    }
+  }
+
+  /**
+   * @method setRequesBody
+   * @summary
+   *
+   * @param requestBody
+   * @returns {Object<Paw.Request>}
+   */
+  private setRequesBody(
+    request: Paw.Request,
+    requestBody: OpenAPIV3.RequestBodyObject,
+  ): Paw.Request {
+    const mediaTypes = Object.keys(requestBody.content)
+
+    if (mediaTypes.length === 0) return request
+
+    const schema = requestBody.content[mediaTypes[0]]
+      .schema as OpenAPIV3.SchemaObject
+
+    request.body = JSON.stringify(jsonSchemaParser(schema, {}) as any, null, 2)
```

Comment:
If you're sure it's a JSON body, you could even use:

```suggestion
    request.jsonBody = jsonSchemaParser(schema, {}) as any
```

Note: `jsonBody` should be an object, so no need for `JSON.stringify()`

Reply:
It's definite the body will always be in [JSON|JSONSchema object](https://swagger.io/specification/#schema-object) (if provided in the OpenAPI v3 document, either as inline or component reference).